template/experiment/research_hub/{% if project_type == 'Basic' %}basic.py{% endif %}

```diff
@@ -0,0 +1,85 @@
+"""
+Basic Workflow
+    Single condition Variable (0-1), Single Observation Variable(0-1)
+    Theorist: LinearRegression
+    Experimentalist: Random Sampling
+    Runner: Firebase Runner (no prolific recruitment)
+"""
+
+from autora.variable import VariableCollection, Variable
+from autora.experiment_runner.firebase_prolific import firebase_runner
+from autora.experimentalist.pipeline import make_pipeline
+import numpy as np
+from sklearn.linear_model import LinearRegression
+from autora.workflow.cycle import Cycle
```

Comment:
nit: Group this autora import with the other autora imports.

Reply:
Copied over from existing AutoRA code. Will address this in a future PR